scalyr_agent/tests/scalyr_logging_test.py

```diff
@@ -240,14 +240,20 @@ def test_rate_limit_no_write_rate(self):
             use_disk=True,
             logs_directory=os.path.dirname(self.__log_path),
             agent_log_file_path=self.__log_path,
-            max_write_burst=250,
+            max_write_burst=500,
             log_write_rate=20000,
         )
         self.__logger = scalyr_logging.getLogger("scalyr_agent.agent_main")
 
         self.__logger.set_keep_last_record(True)
         try:
-            string_300 = "a" * 300
+            # NOTE: Actual value which is being used for the rate limitting is the formatted value
+            # and that value contains much more information than the string we generate here.
+            # This means that 450 + common formatted string data will aways be > 500 and we need to
+            # make sure that "max_write_burst" value we use is large enough so formatted "First
+            # message" and "Second message" (with a warning) fit in that value, otherwise depending
+            # on the timing and fill rate, the test may fail.
+            string_300 = "a" * 450
```

Comment:
Let's change the name to `string_450`.

Reply:
Fixed in 58ba694d1431b37830466261cf4bb2e42db77037.